Source/Fuse.Scripting.JavaScript/ObservableProperty.uno

```diff
@@ -23,11 +24,11 @@ namespace Fuse.Scripting.JavaScript
 
 		This prevents leakage of strong references to observables on the JS side.
 	*/
-	class ObservableProperty: IObserver, IPropertyListener
+	class ObservableProperty: Fuse.Reactive.IObserver, IPropertyListener
```

Comment:
Why do we need to fully qualify this all of a sudden?

Reply:
It collides with `Uno.IObserver`

Reply:
How much worse would it be to drop `using Uno;`? There's a *lot* of qualifying going on here, and it looks to me like this is to be able to write `Uno.Action` and `Uno.Type` four times at the cost of having to add "Fuse.Reactive." 9 times...